app/services/idv/steps/in_person/ssn_step.rb

```diff
@@ -25,6 +25,7 @@ def call
 
           # save the flow path so it's recorded in analytics, eg standard or hybrid flow
           flow_session[:flow_path] = @flow.flow_path
+          current_user
```

Comment:
does this have meaningful side effects? I'm guessing no, so we can delete it?
```suggestion
```

Reply:
removed ... still WIP ... not sure how that worked its wan in there ... thanks 👍🏿